firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteSchemaTest.java

```diff
@@ -307,6 +307,34 @@ public void addsSentinelRows() {
             });
   }
 
+  @Test
+  public void addsSequenceNumberInBatches() {
```

Comment:
Technically this isn't verifying that it does it in batches...  and I think it's similar to the existing addsSentinelRows test?  So would it make sense to call it "addsSentinelRowsForManyDocuments" or something?

Also, probably worth a PORTING NOTE explaining why this test exists (presumably we won't port it to the other platforms).  I'd also call out that while this test exists to verify we can handle lots of documents, it *CANNOT* be used to reproduce the original issue (since it doesn't reproduce with robolectric or whatever).

Reply:
Renamed the test to `addsSentinelRowsForLargeNumberOfDocuments` and added both a porting note and the additional comment that describes the limitations.